Dockerfile

```diff
@@ -53,4 +53,4 @@ COPY ./http/http.env /config/http.env
 COPY ./rest-gw/rest.env /config/rest.env
 RUN sed -ri 's,^([^=]+)=(.*)+$,\1=${\1-\2},' /config/http.env /config/rest.env
 
-ENTRYPOINT ["/config/bin/init-aio.sh"]
+CMD ["/config/bin/init-aio.sh", "true", "true", "true"]
```

Comment:
IIRC that's something very different for the Docker. You can still have `ENTRYPOINT`, but set proper defaults in the script itself. Maybe we can also have these as a set of flags ("nofg", "nohttp", "norest"), it'd be a bit more descriptive even though would require a bit of processing in the script.

Reply:
Replaced to flags. By default, `init-aio.sh` is configured for AIO correct work without passing extra params